rj_config/include/config_server/config_server.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <rclcpp/rclcpp.hpp>
+#include <rj_msgs/msg/field_dimensions.hpp>
+#include <rj_msgs/msg/game_settings.hpp>
+#include <rj_msgs/srv/set_field_dimensions.hpp>
+#include <rj_msgs/srv/set_game_settings.hpp>
+
+namespace config_server {
+using GameSettingsMsg = rj_msgs::msg::GameSettings;
+using SetGameSettingsSrv = rj_msgs::srv::SetGameSettings;
+using SetGameSettingsSrvReqPtr = SetGameSettingsSrv::Request::SharedPtr;
+using SetGameSettingsSrvRespPtr = SetGameSettingsSrv::Response::SharedPtr;
+
+using FieldDimensionsMsg = rj_msgs::msg::FieldDimensions;
+using SetFieldDimensionsSrv = rj_msgs::srv::SetFieldDimensions;
+using SetFieldDimensionsSrvReqPtr = SetFieldDimensionsSrv::Request::SharedPtr;
+using SetFieldDimensionsSrvRespPtr = SetFieldDimensionsSrv::Response::SharedPtr;
+
+class ConfigServer : public rclcpp::Node {
```

Comment:
Please add documentation to `ConfigServer`, `ConfigClient`, and associated functions.

Reply:
Done.